packages/moonstone/Input/Input.module.less

```diff
@@ -53,13 +54,16 @@
 .decorator {
 	display: inline-flex;
 	position: relative;
-	padding: @moon-input-padding (@moon-input-height / 2);
-	border-radius: 1008px;
+	padding: @moon-input-padding;
+	border-radius: @moon-input-border-radius;
 	margin: 0 (@moon-spotlight-outset / 2);
```

Comment:
I think after almost 6 years, it's time to update this margin to match our standard margin. Let's finally convert this to simply use `@moon-spotlight-outset` rather than half of it. Better still would be to make it into a variable: `@moon-input-h-margin: @moon-spotlight-outset`.
With the margin the way it is (and has been), it means when the Input is against the edge of the panel, or vertically next to buttons or other components, it doesn't horizontally line-up.

This 6px margin (originally 5px, and before that 0px) was added here: https://github.com/enyojs/moonstone/commit/3513a05601c400f1802851a0aab6f373b3534e86#diff-2010cd3d397ac502f284f6d890966042R4